app/src/test/java/com/codesoom/assignment/product/domain/ProductTest.java

```diff
@@ -0,0 +1,61 @@
+package com.codesoom.assignment.product.domain;
+
+import com.codesoom.assignment.product.domain.Product;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Product 클래스")
+class ProductTest {
+
+    @Nested
+    @DisplayName("빌더는")
+    class Describe_builder {
+        @Nested
+        @DisplayName("이름이 입력되지 않으면")
```

Comment:
좀 더 구체적으로 표현해줄 수 있겠죠.
```suggestion
        @DisplayName("이름이 null 이면")
```